src/templates/togepi/Togepi.js

```diff
@@ -0,0 +1,262 @@
+import React, { useContext } from 'react';
+
+import { formatDisplayURL } from '../../utils';
+import RichTextArea from '../../shared/RichTextArea';
+import AppContext from '../../context/AppContext';
+
+const Togepi = () =>{
+    const context = useContext(AppContext);
+    const { state } = context;
+    const { data, config, theme } = state;
+
+    const DevideLine = () =>(
+        <div className="grid grid-cols-12" style={{'height':'0.1em'}}>
+                    <div className="col-span-2" style={{background: theme.colors.accent}}/>
+                    <div className="col-span-10" style={{background: theme.colors.primary}}/>
+                </div>
+    );
+
+    const Profile = () => (
+        <div className="flex justify-start">
+            <div className="bg-gray-400">
+                <img className="w-40 h-40" src="https://upload.wikimedia.org/wikipedia/commons/thumb/7/7e/Circle-icons-profile.svg/1200px-Circle-icons-profile.svg.png" alt="profile"/>
```

Comment:
remove the profile image section. A profile image is not applicable in a Canadian resume